src/renderer/Components/Login/index.jsx

```diff
@@ -1,14 +1,132 @@
-    window.location.href = `https://kauth.kakao.com/oauth/authorize?client_id=${import.meta.env.VITE_KAKAO_KEY}&redirect_uri=${import.meta.env.VITE_BASE_URL}&response_type=code`;
+  const navigate = useNavigate();
+  const { setClientStatus } = usePackageStore();
+
+  const [emailValue, setEmailValue] = useState("");
+  const [passwordValue, setPasswordValue] = useState("");
+
+  useEffect(() => {
+    if (window.Kakao && !window.Kakao.isInitialized()) {
+      window.Kakao.init(import.meta.env.VITE_KAKAO_KEY);
+    }
+  }, []);
+
+  const handleLocalLogin = async (event) => {
+    event.preventDefault();
+
+    try {
+      const userCredential = await signInWithEmailAndPassword(
+        auth,
+        emailValue,
+        passwordValue,
+      );
+      const firebaseIdToken = await userCredential.user.getIdToken();
+
+      const response = await axios.post(
+        `${import.meta.env.VITE_SERVER_URL}/auth/sign-in/local`,
+        { firebaseIdToken },
+        {
+          headers: {
+            "Content-Type": "application/json",
+            Authorization: `Bearer ${firebaseIdToken}`,
+          },
+        },
+      );
+
+      const { deliOrderToken, deliOrderRefreshToken, userId, loginType } =
+        response.data;
+      window.localStorage.setItem("deliOrderToken", deliOrderToken);
+      window.localStorage.setItem(
+        "deliOrderRefreshToken",
+        deliOrderRefreshToken,
+      );
+      window.localStorage.setItem("deliOrderUserId", userId);
+      window.localStorage.setItem("deliOrderProvider", loginType);
+
+      setClientStatus({ isLogin: true });
+      navigate("/");
+    } catch (error) {
+      console.error("이메일 로그인 실패: ", error);
+    }
+  };
+
+  const handleGoogleLogin = async () => {
+    try {
+      const provider = new GoogleAuthProvider();
+      const userCredential = await signInWithPopup(auth, provider);
+      const user = userCredential.user;
+
+      const firebaseIdToken = await user.getIdToken();
+
+      const response = await axios.post(
+        `${import.meta.env.VITE_SERVER_URL}/auth/sign-in/google`,
+        { firebaseIdToken },
+        {
+          headers: {
+            "Content-Type": "application/json",
+            Authorization: `Bearer ${firebaseIdToken}`,
+          },
+        },
+      );
+
+      const { deliOrderToken, deliOrderRefreshToken, userId, loginType } =
+        response.data;
+      window.localStorage.setItem("deliOrderToken", deliOrderToken);
+      window.localStorage.setItem(
+        "deliOrderRefreshToken",
+        deliOrderRefreshToken,
+      );
+      window.localStorage.setItem("deliOrderUserId", userId);
+      window.localStorage.setItem("deliOrderProvider", loginType);
+
+      setClientStatus({ isLogin: true });
+
+      navigate("/");
+    } catch (error) {
+      console.error("구글 로그인 실패", error);
+    }
+  };
+  const handleKakaoLogin = async () => {
+    try {
+      const response = await await window.Kakao.Auth.authorize({
```

Comment:
해당 부분에서 await 가 두번 사용 되고 있습니다.😄

```suggestion
      const response = await window.Kakao.Auth.authorize({
```

Reply:
확인하고 처리하도록 하곘습니다!